qutebrowser/mainwindow/tabwidget.py

```diff
@@ -150,6 +150,10 @@ def update_tab_title(self, idx, field=None):
 
         if tabbar.tabText(idx) != title:
             tabbar.setTabText(idx, title)
+
+        # always show only plain title in tooltips
+        title = fields['title']
+        if tabbar.tabToolTip(idx) != title:
```

Comment:
Not sure if we actually need that check - I think you added it for the tab text because `setTabText` recalculates the whole tab bar, right? `setTabToolTip` doesn't do that, though - so I think you can just do `tabbar.setTabToolTip(idx, fields['title'])` unconditionally.

Reply:
ah, I completely forgot I was the one who added that. I probably will add a comment so that dosen't happen again :P